pymobiledevice3/services/mobile_config.py

```diff
@@ -54,6 +69,10 @@ def get_profile_list(self) -> Mapping:
     def install_profile(self, payload: bytes) -> None:
         self._send_recv({'RequestType': 'InstallProfile', 'Payload': payload})
 
+    def install_profile_silent(self, profile: bytes, pkcs12Bytes: bytes, password: str) -> None:
```

Comment:
`pkcs12Bytes` is also a non-PEP8 naming convention.

Reply:
done